app/ui/src/app/app.component.html

```diff
@@ -6,10 +6,7 @@
 
   <ng-template #brandTemplate>
     <a routerLink="/" class="navbar-brand" role="link">
-      <img *ngIf="iconDarkBg && iconDarkBg !== ''" [class]="productBuild ? 'navbar-brand-icon product' : 'navbar-brand-icon'" [src]="loggedIn ? iconDarkBg : iconWhiteBg"
```

Comment:
We do need to keep some of this logic which enables the product branding in production builds.  You can test this locally by changing the `productBuild` setting in your local config.json file to `true`.